data/Language/CSharp.yaml

```diff
@@ -1,7 +1,8 @@
 identifier: CSharp
 full_name: C#
 wikidata: Q2370
-grammar_file: https://github.com/antlr/grammars-v4/tree/master/csharp/CSharpParser.g4
+grammar_file: https://github.com/antlr/grammars-v4/tree/master/csharp/
+CSharpParser.g4
 aliases:
```

Comment:
syntax error: could not find expected ':'

*Origin: YAMLLintBear, Section: `yaml`.*